Add documentation for the new Hypontech Cloud integration (#42730)
home-assistant/home-assistant.github.io: source/_integrations/hypontech.markdown

---
title: Hypontech Cloud
description: Instructions on how to integrate Hypontech Cloud solar inverters within Home Assistant.
ha_category:
  - Energy
  - Sensor

> ha_release: 2026.1

Reviewer: ```suggestion
ha_release: 2026.2
```

> ha_release: 2026.2

Reviewer 2: ```suggestion
ha_release: 2026.3
```

ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@jcisio'
ha_domain: hypontech
ha_platforms:
  - sensor
ha_integration_type: device
ha_quality_scale: bronze
---

The **Hypontech Cloud** {% term integration %} allows you to monitor your Hypontech solar inverter system through the [Hypontech Cloud](https://www.hypon.cloud) platform and integrate the data into your Home Assistant installation.

## Supported devices

This integration supports all Hypontech inverters and micro inverters that are connected to the Hypontech Cloud platform.

## Prerequisites

To use this integration, you need:

- A Hypontech Cloud account with access to your solar inverter system
- Your Hypontech Cloud username and password

{% include integrations/config_flow.md %}

To set up the integration, you need the following information:

{% configuration_basic %}
Username:
  description: "Your Hypontech Cloud account username."
Password:
  description: "Your Hypontech Cloud account password."
{% endconfiguration_basic %}

## Supported functionality

### Sensors

> The integration provides one **Plant** device for each location (e.g. Balcony, Garden, Home, Office) and one **Overview** (aggregation) device. Each has the following sensors:

Reviewer: ```suggestion
The integration provides one **Plant** device for each location (for example: Balcony, Garden, Home, Office) and one **Overview** (aggregation) device. Each has the following sensors:
```
we prefer "for example", or "such as" over e.g., see also https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg

---
title: Hypontech Cloud
description: Instructions on how to integrate Hypontech Cloud solar inverters within Home Assistant.
ha_category:
  - Energy
  - Sensor
ha_release: 2026.1
ha_iot_class: Cloud Polling
ha_config_flow: true
ha_codeowners:
  - '@jcisio'
ha_domain: hypontech
ha_platforms:
  - sensor
ha_integration_type: device
ha_quality_scale: bronze
---

> The **Hypontech Cloud** {% term integration %} allows you to monitor your Hypontech solar inverter system through the Hypontech Cloud platform and integrate the data into your Home Assistant installation.

Reviewer: could you add a link to the related Hypontech Cloud website?

Author: Done.

## Supported devices

This integration supports all Hypontech inverters and micro inverters that are connected to the Hypontech Cloud platform.

## Prerequisites

To use this integration, you need:

- A Hypontech Cloud account with access to your solar inverter system
- Your Hypontech Cloud username and password

{% include integrations/config_flow.md %}

To set up the integration, you need the following information:

{% configuration_basic %}
Username:
  description: "Your Hypontech Cloud account username."
Password:
  description: "Your Hypontech Cloud account password."
{% endconfiguration_basic %}

## Supported functionality

### Sensors

The integration provides the following sensors:

> | Name | Unit | Description |

Reviewer: Although this table is probably small enough to fit, we [prefer lists over tables](https://developers.home-assistant.io/docs/documenting/general-style-guide#tables), see [example](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#L105).

Author: Done.

|------|------|-------------|

> | Power | W | Current power production from your solar system |

Reviewer: ```suggestion
**Power** (W):  Current power production from your solar system
```
You could use a list style like this, for example.
